compiler/qsc_parse/src/item.rs

```diff
@@ -127,6 +133,7 @@ fn parse_namespace(s: &mut Scanner) -> Result<Namespace> {
     let name = dot_ident(s)?;
     token(s, TokenKind::Open(Delim::Brace))?;
     let items = barrier(s, &[TokenKind::Close(Delim::Brace)], parse_many)?;
+    check_empty_namespace(s, items.as_ref());
```

Comment:
I'm not sure I agree that empty namespace should be a parse error, it's perfectly valid syntax.

Reply:
Hey @minestarks thanks for clarifying.

The example code in the issue threw me off a bit. Since removing the doc comment also produced an error, I assumed errors for empty namespaces was intentional. Working on a new solution now with your feedback in mind.